netbox/extras/api/serializers_/notifications.py

```diff
@@ -0,0 +1,82 @@
+from drf_spectacular.utils import extend_schema_field
+from rest_framework import serializers
+
+from core.models import ObjectType
+from extras.models import Notification, NotificationGroup, Subscription
+from netbox.api.fields import ContentTypeField, SerializedPKRelatedField
+from netbox.api.serializers import ValidatedModelSerializer
+from users.api.serializers_.users import GroupSerializer, UserSerializer
+from users.models import Group, User
+from utilities.api import get_serializer_for_model
+
+__all__ = (
+    'NotificationSerializer',
+    'NotificationGroupSerializer',
+    'SubscriptionSerializer',
+)
+
+
+class NotificationSerializer(ValidatedModelSerializer):
+    object_type = ContentTypeField(
+        queryset=ObjectType.objects.with_feature('notifications'),
+    )
+    object = serializers.SerializerMethodField(read_only=True)
+    user = UserSerializer(nested=True)
+
+    class Meta:
+        model = Notification
+        fields = [
+            'id', 'url', 'display', 'object_type', 'object_id', 'object', 'user', 'created', 'read', 'event_type',
```

Comment:
should there be a display_url here as well

Reply:
There's no dedicated UI view for a notification or subscription, but I'll add `display_url` for notification groups.